config/peerpods/podvm/lib.sh

```diff
@@ -313,6 +336,41 @@ EOF
 
 }
 
+# Function to split image type, url and path from PODVM_IMAGE_URI for pre-built image scenario.
+function get_image_type_url_and_path() {
+
+    # Use awk to split on '::' and then on ':', and capture output
+    read -r PODVM_IMAGE_TYPE PODVM_IMAGE_URL PODVM_IMAGE_TAG PODVM_IMAGE_SRC_PATH < <(
```

Comment:
I'd allow to empty image path and use the default /tmp/podvm.qcow2 , or empty tag, so that `<image_type>::<image_url>:<image_tag>`, `<image_type>::<image_url>::<image_path>` and `<image_type>::<image_url>` will be valid, if possible 
Impressive awk skills BTW :)

Reply:
incorporated..